src/solver/CMakeLists.txt

```diff
@@ -46,8 +46,9 @@ OMESSAGE("  :: application solver")
 
 set(exec_name "antares-solver")
 
-add_library(solver-lib INTERFACE)
-
+add_library(solver-lib INTERFACE
+        ${SRCS}
+)

```

Comment:
I have a feeling this will add the icon to solver-lib (see l. 26-42), when what we really want is adding to executable antares-solver. Am I wrong ?